scripts/install-build-deps

```diff
@@ -145,7 +145,21 @@ SEAGATE_GITHUB_CORTX_DEPS_REPO=https://github.com/Seagate/cortx/releases/downloa
 if ! $use_ansible ; then
   case $(distro_type) in
       redhat)
-        yum-config-manager --enable powertools
+        source /etc/os-release
+        if [[ $NAME == "Rocky Linux" ]];
+        then
+	    if [[ ! `yum repolist | grep Power` ]]
+            then
+                yum-config-manager --enable powertools
+                yum -y install epel-release
+            fi
+	elif [[ $NAME == "Oracle Linux Server" ]]
+	then
+                yum-config-manager --enable ol8_codeready_builder
+        else
+		yum -y install epel-release
+	fi
+	echo "Make sure that epel-release repo is present.."
```

Comment:
Shouldn't this go before 1st if?

Reply:
This comment is added for Oracle Linux and other OSes.